app/src/main/java/fr/free/nrw/commons/MediaDataExtractor.java

```diff
@@ -23,6 +25,7 @@
     private final MediaWikiApi mediaWikiApi;
     private final OkHttpJsonApiClient okHttpJsonApiClient;
     private final MediaClient mediaClient;
+    private String depiction, caption;
```

Comment:
multiple depictions

Reply:
@nicolas-raoul for multiple depictions I have added a new line this to https://github.com/commons-app/apps-android-commons/blob/2107a60abe3d8b0aaa3780c1ab169ae0cab5b6ea/app/src/main/java/fr/free/nrw/commons/media/MediaClient.java#L195
separate this depictions when they are displayed on text view